scripts/package

```diff
@@ -14,7 +14,7 @@ REPO=${REPO:-rancher}
 
 cp ../bin/rancher-vxlan .
 
-IMAGE=${REPO}/vxlan:${TAG}
+IMAGE=${REPO}/rancher-vxlan:${TAG}
```

Comment:
I think based on the discussion on Slack few minutes back, we don't need to generate this image at all. We will package this binary  as part of `rancher/net`. So just generating a .tar.gz file with the binary is sufficient.